src/PageLayout/Notification/Recurring/RecurringDeliveryRequestController.js

```diff
@@ -0,0 +1,94 @@
+import React, { Component } from 'react';
+import '../Popup.css';
+
+import RecurringDeliveryRequest1 from './RecurringDeliveryRequest1';
+import RecurringDeliveryRequest2 from './RecurringDeliveryRequest2';
+import RecurringDeliveryRequest3 from './RecurringDeliveryRequest3';
+import EnterPrimaryContact from './EnterPrimaryContact';
+import close from '../../../icons/cross-out.svg';
+
+class RecurringDeliveryRequestController extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            step: 1
+        };
+    }
+
+    enterPrimaryContact(){
+        this.setState({
+            step: 4
+        });
+    }
+
+    savePrimaryContact(){
+        // TODO: save fields to DB
+        this.setState({
+            step: 2
+        });
+    }
+
+    nextStep() {
+        this.setState((prevState) => {
+            return { step: prevState.step + 1 };
+        });
+    }
+
+    previousStep() {
+        this.setState((prevState) => {
+            return { step: prevState.step + 1 };
+        });
+    }
+
+    showStep() {
+        switch (this.state.step) {
+        default:
+            return <RecurringDeliveryRequest1
```

Comment:
nit-pick: could we maybe name the components to be a little more informative? So something like `RecurringRequestDetails`, `RecurringRequestDisclaimer`, `RecurringRequestClaimed`